### PR TITLE
allow custom option flags

### PR DESCRIPTION
**Pull request by alacy-alteryx**

### CODE REVIEW

File: resources/package.rb [hunk 1]
```diff
@@ -18,17 +19,24 @@ def package_name
...
 
 action :install do
   pkg_name = package_name
   pkg_source = source
   pkg_version = version
   pkg_timeout = timeout
+  pkg_options = options
+
+  # Allow custom options but make sure the silent flag is in the options list
+  unless pkg_options =~ /\/s/
```
Comment (jakemig) [inline resources/package.rb]: Can we change this to something like `pkg_options += ' /s' unless pkg_options.include?('s/')`?

Comment (alacy-alteryx) [reply resources/package.rb]: Updating with the next pull request

File: metadata.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 maintainer       'Alteryx, Inc.'
 maintainer_email ''
-license          'Apache 2.0'
+license          'Apache-2.0'
 description      'Installs/Configures Alteryx server'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.13.9'
+version          '0.13.10'
 supports         'windows'
+chef_version     '~> 12'
```
Comment (jakemig) [inline metadata.rb]: As per the readme, let's set this to `>= 12.7.2`.

File: resources/package.rb [hunk 3]
```diff
@@ -18,17 +19,22 @@ def package_name
...
 
 action :install do
   pkg_name = package_name
   pkg_source = source
   pkg_version = version
   pkg_timeout = timeout
+  pkg_options = options
+
+  # Allow custom options but make sure the silent flag is in the options list
+  pkg_options += ' /s' unless pkg_options.include?('s/')
```
Comment (jakemig) [inline resources/package.rb]: `include?('s/')` => `include?('/s')`

**Pull request merged**